feature/quicksettings/src/main/java/com/google/jetpackcamera/feature/quicksettings/QuickSettingsScreen.kt

```diff
@@ -81,6 +82,9 @@ fun QuickSettingsScreenOverlay(
         )
 
     if (isOpen) {
+        BackHandler {
+            toggleIsOpen()
```

Comment:
Do we want to add something similar for expanded quick settings? Rather than toggling `isOpen`, should we return to the non-expanded state?

Reply:
done